bin/update-ips.bash

```diff
@@ -92,11 +92,10 @@ yq_eval() {
 
   if ${dry_run}; then
     log_warning "Diff found for ${config_option} in ${config_filename} (diff shows actions needed to be up to date)"
+    has_diff=$((has_diff + 1))
   else
     yq4 -i "${expression}" "${config_file}"
   fi
-
-  has_diff=$((has_diff + 1))
```

Comment:
Is there a reason why that is conditional here, rather than the final exit statement?

Reply:
I'm not sure I follow. Do you mean why it's not exiting right then and there? If so then I think it's because there can be multiple diffs.

Reply:
No, I'm wondering why we make the counting optional, rather than the exit with the count?
It make more sense to me to have the condition on the final exit statement to not propagate the count rather than make counting optional and still exit on the count.